plugins/techdocs-backend/src/service/router.ts

```diff
@@ -87,10 +87,17 @@ export async function createRouter(
   const { publisher, config, logger, discovery } = options;
   const catalogClient = new CatalogClient({ discoveryApi: discovery });
 
+  // Entities are cached to optimize the /static/docs request path, which can be called many times
+  // when loading a single techdocs page.
+  const entityLoader = new CachedEntityLoader({
+    catalog: catalogClient,
+    cache: options.cache.getClient(),
+  });
+
   // Set up a cache client if configured.
   let cache: TechDocsCache | undefined;
   const defaultTtl = config.getOptionalNumber('techdocs.cache.ttl');
-  if (options.cache && defaultTtl) {
+  if (defaultTtl) {
     const cacheClient = options.cache.getClient({ defaultTtl });
     cache = TechDocsCache.fromConfig(config, { cache: cacheClient, logger });
```

Comment:
would the cache not get set if there is no `defaultTtl`?

Reply:
oh i see but the entity cache and the techdocs cache are kept separate, never mind!